loop/routes/rooms.js

```diff
@@ -385,17 +385,22 @@ module.exports = function (apiRouter, conf, logError, storage, filestorage, auth
                   participants) {
                     if (res.serverError(err)) return;
 
+                    // The room participants list except ourselves.
+                    var otherParticipants = participants.filter(function(participant) {
```

Comment:
Add a comment stating that "otherParticipants" is the list of non-connected participants.